From: Debian Rust Maintainers <pkg-rust-maintainers@alioth-lists.debian.net>
Date: Thu, 13 Jun 2024 11:16:39 +0200
Subject: d-ignore-test_arc_condvar_poison-ppc

Forwarded: no
---
 library/std/tests/sync/mutex.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/library/std/tests/sync/mutex.rs b/library/std/tests/sync/mutex.rs
index 75a6bf6..65019f9 100644
--- a/library/std/tests/sync/mutex.rs
+++ b/library/std/tests/sync/mutex.rs
@@ -424,6 +424,7 @@ fn test_replace_poison() {
 }
 
 #[test]
+#[cfg(not(target_arch = "powerpc"))]
 #[cfg_attr(not(panic = "unwind"), ignore = "test requires unwinding support")]
 fn test_mutex_arc_poison() {
     let arc = Arc::new(Mutex::new(1));
