From: Debian Rust Maintainers <pkg-rust-maintainers@alioth-lists.debian.net>
Date: Thu, 13 Jun 2024 11:16:39 +0200
Subject: d-ignore-test_arc_condvar_poison-ppc

Forwarded: no
---
 library/std/tests/sync/mutex.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/library/std/tests/sync/mutex.rs b/library/std/tests/sync/mutex.rs
index 90cefc0..388d377 100644
--- a/library/std/tests/sync/mutex.rs
+++ b/library/std/tests/sync/mutex.rs
@@ -458,6 +458,7 @@ fn test_replace_poison() {
 }
 
 #[test]
+#[cfg(not(target_arch = "powerpc"))]
 #[cfg_attr(not(panic = "unwind"), ignore = "test requires unwinding support")]
 fn test_arc_condvar_poison() {
     struct Packet<T>(Arc<(Mutex<T>, Condvar)>);
